ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3SubsystemAdd.java

```diff
@@ -475,10 +474,6 @@ protected void execute(DeploymentProcessorTarget processorTarget) {
             EJB3SubsystemDefaultPoolWriteHandler.SLSB_POOL.updatePoolService(context, model);
         }
 
-        if (model.hasDefined(DEFAULT_ENTITY_BEAN_INSTANCE_POOL)) {
-            EJB3SubsystemDefaultPoolWriteHandler.ENTITY_BEAN_POOL.updatePoolService(context, model);
-        }
-
         if (model.hasDefined(DEFAULT_SFSB_CACHE)) {
```

Comment:
below around line 502, any reason the similar is not applied to `DEFAULT_ENTITY_BEAN_OPTIMISTIC_LOCKING`?

Reply:
It seems to me that DEFAULT_ENTITY_BEAN_OPTIMISTIC_LOCKING has some effect so I thought it would be better if we remove it the same time those deprecated attributes are.